ui/shared/components/panel/variants/Frequencies.jsx

```diff
@@ -182,72 +202,125 @@ const POPULATIONS = [
     queryParams: { [GENOME_VERSION_37]: 'dataset=gnomad_sv_r2_1' },
     helpMessage: GNOMAD_SV_CRITERIA_MESSAGE,
   },
+]
+
+const MITO_POPULATIONS = [
+  {
+    field: 'callset',
+    fieldTitle: 'This Callset',
+    acDisplay: 'AC',
+    titleContainer: sectionTitle,
+    section: HOM_SECTION,
+  },
+  {
+    field: 'callset_heteroplasmy',
+    fieldTitle: 'This Callset',
+    acDisplay: 'AC',
+    titleContainer: sectionTitle,
+    section: HET_SECTION,
+    precision: 3,
```

Comment:
use the same precision for callset and callset_heteroplasmy, there is no reason for them to be different

Reply:
Good catch. It shouldn't have `precision`.